waku/v2/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -62,7 +62,8 @@ type
 const DefaultKeyStorePath* = "rlnKeystore.json"
 const DefaultKeyStorePassword* = "password"
 
-const BlockChunkSize* = 100'u64
+const DecayFactor* = 1.2
```

Comment:
Is this decay for block chunk size added by mistake to this PR? Does not seem to be really related to "close db connection" (or am I missing something?)

Reply:
unfortunately, while I was testing this fix, I discovered a bug related to the chunking of blocks, so I included it in this PR to ensure some stability - can remove if you'd like!

Reply:
Fine by me, just wanted to make sure this was intentional:)